addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -692,30 +692,29 @@ function factory(dependencies) {
         _notifyNewChannelMessageWhileOutOfFocus({ channel, message }) {
             const author = message.author;
             const messaging = this.env.messaging;
-            let notificationTitle;
+            let notificationHeader;
             if (!author) {
-                notificationTitle = this.env._t("New message");
+                notificationHeader = this.env._t("New message");
             } else {
                 const authorName = author.nameOrDisplayName;
                 if (channel.channel_type === 'channel') {
-                    const groupIcon = '<i class="fa fa-hashtag"/> ';
-                    const privateIcon = '<i class="fa fa-lock"/> ';
-                    const channelIcon = channel.public === 'private' ? privateIcon : groupIcon;
+                    const channelIcon = channel.public === 'private' ? '<i class="fa fa-lock"/> '
+                                                                     : '<i class="fa fa-hashtag"/> ';
                     const channelName = owl.utils.escape(channel.displayName);
                     const channelNameWithIcon = channelIcon + channelName;
-                    notificationTitle = _.str.sprintf(
-                        this.env._t("<b>%s from %s</b>"),
+                    notificationHeader = _.str.sprintf(
+                        this.env._t("%s from %s"),
                         owl.utils.escape(authorName),
                         channelNameWithIcon
                     );
                 } else {
-                    notificationTitle = _.str.sprintf(
-                        this.env._t("<b>%s</b>"),
-                        owl.utils.escape(authorName)
-                    );
+                    notificationHeader = owl.utils.escape(authorName);
                 }
             }
-            const notificationContent = notificationTitle + message.prettyBody.substr(0, PREVIEW_MSG_MAX_SIZE);
+            document.body.classList.add('o_notification_discuss');
```

Comment:
That's not ok, why didn't you keep the previous padding change in /web?

Reply:
--> Because there is no bottom space between the content and line.
-->As you can see the picture in specification(https://drive.google.com/file/d/1iV2dVLDO8L3QobMbqJ2QvwLpDxtFNTfO/view?usp=sharing) 
-->There is a space in bottom.
--> If we keep padding from /web then it looks like this.
![Screenshot from 2020-08-14 15-20-16](https://user-images.githubusercontent.com/54100921/90237739-6318d100-de42-11ea-9133-b959ef2de181.png)
--> Is it okay for you? If it is yes then i will remove this and keep old change.

Reply:
@seb-odoo Could you please look into this? Thanks..